### DISCUSSION TITLE
Turn usb mouse to bluetooth mouse

### DISCUSSION THREAD

**Discussion by Chandandada** [AppGround-io/bluetooth-keyboard-and-mouse-support#241] [Q&A] [answered]
I want to use my mechanical usb keyboard and usb mouse as bluetooth keyboard and mouse for my Android tab. For which I am using my another Android phone in which the usb keyboard and mouse is attached and installed your Serverless keyboard and mouse  app. It is working good. 
I have a problem for which I am asking here.
The mouse pointer does not respond until the mechanical mouse is pressed ( left button) and moved within the displayed mouse pad area. 

I want
1. the mouse pointer hover freely according to the movement of the mechanical mouse without pressing any button. 
2. The mouse pointer works beyond the specified pad area.

Please suggest a solution. I need to make clear these things before I purchase the premium version. 

Thanks.

**Comment by GigaZelensky** [comment]
YES! PLEASEEEE i have been waiting for a feature like this for a long time

**Comment by AppGround-io** [comment, marked as answer]
This feature is included in the upcoming version 4.16.0 of the app.
